File: Code/cpp/prime.cpp
#include<iostream>
using namespace std;
int main()
{
    int num, i;
    string prime = "Prime";

    cout << "Enter Number: ";
    cin >> num;

    for (i = 2; i < num; i++)
    {
        if (num % i == 0)
        {
            prime.replace(0, 10, "Not Prime");
            break;
        }
    }
    cout << prime;
    return 0;
}

File: Code/java/prime.java
import java.util.Scanner;

class Prime
{
    public static void main(String[] args)
    {
        int num, i;
        String prime = "Prime";
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Number: ");
        num = scanner.nextInt();

        for (i = 2; i < num; i++)
        {
            if (num % i == 0)
            {
                prime = "Not Prime";
                break;
            }
        }
        System.out.println(prime);

        scanner.close();
    }
}